### PR TITLE
Adapt to `yiisoft/middleware-dispatcher` changes

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | yiisoft/middleware-dispatcher#56

### CODE REVIEW

File: tests/GroupTest.php [hunk 1]
```diff
@@ -445,7 +446,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
     private function getDispatcher(): MiddlewareDispatcher
     {
         return new MiddlewareDispatcher(
-            new MiddlewareFactory(new Container([])),
+            new MiddlewareFactory(new Container([]), new WrapperFactory(new Container([]))),
```
Comment (arogachev) [inline tests/GroupTest.php:449]:
```suggestion
            new MiddlewareFactory(new Container([]), 
            new WrapperFactory(new Container([]))),
```

Comment (arogachev) [inline tests/GroupTest.php:449]: Lots of parentheses - quite hard to read. Maybe refactor using assignments to separate variables

**Review by arogachev — CHANGES_REQUESTED** [review]
:+1:

**Comment by arogachev** [comment]
Need to update middleware dispatcher dependency - https://github.com/yiisoft/middleware-dispatcher/releases/tag/3.0.0.

File: tests/GroupTest.php (hunk 1 above)
Comment (rustamwin) [reply tests/GroupTest.php]: Done.

**Comment by rustamwin** [comment]
> Need to update middleware dispatcher dependency - https://github.com/yiisoft/middleware-dispatcher/releases/tag/3.0.0.

Done.

**Review by arogachev — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**